routes/book.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+
+const router = express.Router();
+
+const bookController = require('../controllers/bookController')
+
+//var books=[];
+
+router.get("/", bookController.getAllBook);
+
```

Comment:
get request which fetches all the items should have plural naming. getAllBooks in this case